models/index.js

```diff
@@ -1,10 +1,12 @@
 const mongoose = require('mongoose');
 require('dotenv').config();
 // TODO: include all model files here (and export models together below)
-const testModels = require('./test');
+
+const projectModel = require(`./project.js`);
+const developerModel = require(`./developer.js`);
 
 // connect to Mongo DB
-mongoose.connection.openUri(process.env.MONGODB_URI || process.env.DB_CONN, {}, function(err, conn) {
+mongoose.connection.openUri(process.env.MONGODB_URI || "https://sheltered-taiga-43222.herokuapp.com/", {}, function(err, conn) {
```

Comment:
This probably isn't what you intended to do. This function expects the location of your Mongo DB, so pointing it to a Node server on Heroku probably won't work as intended.

Instead, you could give it a different environment variable to fall back on.